test/exe/pie_test.sh

```diff
@@ -13,6 +13,10 @@ if readelf -hW "${ENVOY_BIN}" | grep "Type" | grep -o "DYN (Shared object file)"
   echo "${ENVOY_BIN} is a PIE!"
   exit 0
 fi
+if readelf -hW "${ENVOY_BIN}" | grep "Type" | grep -o "DYN (Position-Independent Executable file)"; then
```

Comment:
fI build Envoy with some target(fastbuild, debug) at Linux, all of them are DYN shared object, which case would it belong to Position-Independent?

Reply:
I guess it is environment dependent? I haven't seen this before though.